### ISSUE TITLE
Scenario 2b - Syntax error in the AMR

### ISSUE THREAD

**Issue by VigneshSella** [indralab/mira#213]
In the [AMR](https://github.com/indralab/mira/blob/hackathon/notebooks/hackathon_2023.07/scenario2_2_b_multi_vax_multi_strain.json) there is a syntax error in the rate expression "t1" (and I believe most of the expressions):
```
{"ode": {"rates": [{"target": "t1", "expression": "I_cido:0000371*V1_vo:0004987*ai*beta_0*beta_scale"
```
The terms `"I_cido:0000371` and `V1_vo:0004987` are syntax errors and includes the ID I believe.

**Comment by VigneshSella** [comment]
I think this should help as well:
https://github.com/indralab/mira/blob/5f22de7587de3ee8a484a60c3284448e5155e9fb/mira/metamodel/utils.py#L7

**Comment by bgyori** [comment]
I see, I didn't realize that these couldn't be parsed after being generated. I'll try to come up with a workaround and regenerate the models.

**Comment by bgyori** [comment]
Should be fixed in 40d0f4a

**Comment by VigneshSella** [comment]
Fixed ^
